spin/cmds/meson.py

```diff
@@ -499,9 +499,15 @@ def run(ctx, args):
     default=True,
     help="Build project before generating docs",
 )
+@click.option(
+    "--plot/--no-plot",
+    "sphx_gallery_plot",
```

Comment:
I'm just in extreme bikeshed mode, so feel free to ignore this: why do you save 2 characters here rather than be fully explicit with `sphinx`? (I fully agree that it's unambiguous even with `sphx`, but I feel it is more a jargon than when it's spelt out fully)

Reply:
Fixed.